### DISCUSSION TITLE
Hide Agent's Hand

### DISCUSSION THREAD

**Discussion by notnil** [SirRender00/texasholdem#170] [Q&A] [answered]
I was going to implement an agent but I actually want to play against them.  Is there a way to hide their hand?

**Comment by notnil** [comment]
It's also not immediate clear how to run the GUI in this case.

**Comment by SirRender00** [comment, marked as answer]
Does this section of the docs answer your question? https://texasholdem.readthedocs.io/en/stable/guis.html#playing-with-agents

**Comment by notnil** [comment]
Missed the docs, thanks!
